### DISCUSSION TITLE
SMR rewards disappeared

### DISCUSSION THREAD

**Discussion by Buenos32** [iotaledger/firefly#7662] [Help & Support]
I was not able to migrate my SMR from Firefly wallet to Firefly SMR wallet. It was in my firefly but upon the decision to convert assembly rewards to IOTA, it totally disappeared. Do not know, what to do.

**Comment by Flatboy99** [comment]
Same happened to me - all my shimmer showed up on Firefly separate from my IOTA and Assembly tokens, but since I did the latest update my shimmer is gone???? I originally didn't move my shimmer to Firefly Shimmer as it was having issues at the time.  How do I find where my Shimmer went??

**Comment by begonaalvarezd** [comment]
Quick information about ASMB and SMR:

- Assembly: please read https://blog.iota.org/stardust-upgrade-iota-tokenomics/. The assembly project does not exist anymore, but your rewards got airdropped to you in form of IOTAs. They will appear in the new version 2.0.3 in the vesting tab. If you are missing balances you can run the balance finder from settings > advanced > balance finder, or if you are missing rewards you can also run the rewards finder from the vesting tab, the '....' menu, search vesting rewards
- Shimmer: what the old firefly version displayed was just a representation of what you owned, but you couldnt control the shimmers from the iota app. You must download the firefly shimmer app to control those funds, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/

Will close the issue because the new firefly app is not intended to display old staking rewards, but the new iota airdrop is on the new app and the shimmer tokens should be managed from the shimmer app. 

Please let us know if you have any other issues

**Comment by Flatboy99** [comment]
Thanks kindly, I followed your advice and was able to find my Shimmer.

Take care.  Marc
